src/app/static/src/app/store/baseline/actions.ts

```diff
@@ -111,18 +111,18 @@ export const actions: ActionTree<BaselineState, DataExplorationState> & Baseline
                 .then((response) => {
                     if (response) {
                         const metadata = response.data;
-                        const avaliableResources = rootGetters ? rootGetters["baseline/selectedDatasetAvailableResources"] : {}
+                        const availableResource = rootGetters ? rootGetters["baseline/selectedDatasetAvailableResources"] : {}
```

Comment:
I agree with fixing the typo in `avaliable` but I don't think switching to singular for the `Resource` bit is helpful, since this object describes all resources. 
```suggestion
                        const availableResources = rootGetters ? rootGetters["baseline/selectedDatasetAvailableResources"] : {}
```